activestorage/lib/active_storage/attached/many.rb

```diff
@@ -17,6 +17,10 @@ def blobs
       change.present? ? change.blobs : record.public_send("#{name}_blobs")
     end
 
+    def persisted_blobs #:nodoc:
+      blobs.select { |blob| blob.persisted? }
+    end
```

Comment:
Can we make this method private?

Reply:
We won't be able to change this to a private method 😊 .

Reply:
I think `:nodoc:` is sufficient to make it Private API (even with public visibility) for documentation purposes.